docs/logging.rst

```diff
@@ -32,6 +32,7 @@ Access log by default is switched on and uses ``'aiohttp.access'``
 logger name.
 
 The log may be controlled by :meth:`aiohttp.web.Application.make_handler` call.
+.. deprecated:: 3.2
```

Comment:
Please remove the deprecation but replace `aiohttp.web.Application.make_handler` with `aiohttp.web.AppRunner` / `aiohttp.web.run_app()`